Azure/azure-cli
# [Compute] New command: vm install-patches

**Description**

New command vm install-patches
Resolve #16790 

***Example***
az vm install-patches -g MyResourceGroup -n MyVm --maximum-duration PT4H --reboot-setting IfRequired --classifications-to-include Critical Security --exclude-kbs-requiring-reboot true

**Testing Guide**

``` python
        self.cmd(
            'vm create -g {rg} -n vm --image Win2019Datacenter --enable-hotpatching true --admin-username azureuser --admin-password testPassword0 --nsg-rule NONE')
        self.cmd('vm install-patches -g {rg} -n vm --maximum-duration PT4H --reboot-setting IfRequired '
                 '--classifications-to-include Critical Security --exclude-kbs-requiring-reboot true', checks=[
            self.check('status', 'Succeeded')
        ])
```
**History Notes**


---

This checklist is used to make sure that common guidelines for a pull request are followed.
